### PR TITLE
Style Node is inserted multiple times

### LINKED ISSUE #1510: file-upload: adding style-nodes multiple times
Reproducer:
- open axa-file-upload
- upload a valid file
- delete it
- a style-node is inserted in dom every time you delete a file

image

### PR DESCRIPTION
**Pull request by danyball**
Fixes #1510 

# Done is when (DoD):
- [ ] I have added UI/Unit tests for my changes
- [ ] Design has been reviewed with (here name of reviewer)
- [ ] Old tests and eslint rule are not broken
- [x] My changes are well documented in the README and showcased in the stories 
- [x] Tested with IE

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas (focus on **WHY**)
- [ ] I have made/updated the ChangeLog Section in the README

### CODE REVIEW

**Review by raphaellueckl — DISMISSED** [review]
File removal is weird

**Comment by danyball** [comment]
I need to fix IE

**Review by LucaMele — DISMISSED** [review]
did u test it in IE?

File: CONTRIBUTION.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ it('should sum numbers', () => {
 - Never and in any circumstances use `==` over `===`
 - Enumerate as many falsy values in your Boolean checks as possible and be aware that `0` and `""` are also falsy.
 - The [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) is a very important principle. But even more important is to know exactly when to break it. Sometimes a super DRY climate can lead to over-engineered solutions that end up being unmaintainable :grinning:. There is no rule here, just be careful to not implement a system that is too complex. Lots of options in interfaces or configuration possibilities could be one indicator that the system gets too complex.
+- Always clean up if a component is destroyed. If you add something in the DOM outside the component you have to remove that on destroy.
```
Comment (markus-walther) [inline CONTRIBUTION.md]: better: If you have side effects  outside of your component (e.g. global event listeners, added DOM nodes ) you have to undo those side effects in `disconnectedCallback`.

File: src/utils/inline-styles.js [hunk 2]
```diff
@@ -59,6 +61,9 @@ export default class InlineStyles extends LitElement {
     // remove or update reference-count metadata
     if (referenceCount < 1) {
       referenceCounts.delete(root); // give an early hint to GC
+      if (styleTagNode && styleTagNode.parentNode) {
+        styleTagNode.parentNode.removeChild(styleTagNode); // remove style node as well
+      }
```
Comment (markus-walther) [inline src/utils/inline-styles.js:66]: Good! Clean and defensively programmed...

**Review by markus-walther — CHANGES_REQUESTED** [review]
Small wording improvements, otherwise LGTM

**Review by markus-walther — DISMISSED** [review]
LGTM

File: CONTRIBUTION.md [hunk 3]
```diff
@@ -89,6 +89,7 @@ it('should sum numbers', () => {
 - Never and in any circumstances use `==` over `===`
 - Enumerate as many falsy values in your Boolean checks as possible and be aware that `0` and `""` are also falsy.
 - The [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) is a very important principle. But even more important is to know exactly when to break it. Sometimes a super DRY climate can lead to over-engineered solutions that end up being unmaintainable :grinning:. There is no rule here, just be careful to not implement a system that is too complex. Lots of options in interfaces or configuration possibilities could be one indicator that the system gets too complex.
+- If you have side effects outside of your component (e.g. global event listeners, added DOM nodes ) you have to undo those side effects in disconnectedCallback
```
Comment (raphaellueckl) [inline CONTRIBUTION.md]: `added DOM nodes) you`

one whitespice too much!

Comment (LucaMele) [reply CONTRIBUTION.md]: very white 🌶️

**Review by LucaMele — APPROVED** [review]
LGTM

**Pull request merged**